cf_remote/ssh.py

```diff
@@ -183,22 +183,22 @@ def scp(file, remote, connection=None, rename=None, hide=False):
         with connect(remote) as connection:
             scp(file, remote, connection, rename, hide=hide)
     else:
-        print_function = log.debug if hide else print
-        print_function("Copying: '%s' to '%s'" % (file, remote))
+        if not hide:
+            print("Copying: '%s' to '%s'" % (file, remote))
```

Comment:
Not a big deal, but you have changed the logging in a not-so-obvious way: Before it was only logged to one place: either `print`, or `log.debug`. Now, after your change, it's printed twice (always to log.debug, and sometimes to print).